Tests/src/PublicAPITests/OrganizationSystemTests.cpp

```diff
@@ -59,16 +85,16 @@ void GetUsersRoles(::OrganizationSystem* OrganizationSystem, const csp::common::
 InviteOrganizationRoleCollection CreateOrganizationInvites()
 {
 	InviteOrganizationRoleInfo InviteUser1;
-	InviteUser1.UserEmail = AltUser1NonMemberEmail;
-	InviteUser1.OrganizationRoles  = {EOrganizationRole::Member};
+	InviteUser1.UserEmail		  = DefaultLoginEmail;
+	InviteUser1.OrganizationRoles = {EOrganizationRole::Member};
 
 	InviteOrganizationRoleInfo InviteUser2;
-	InviteUser2.UserEmail = AltUser2NonMemberEmail;
-	InviteUser2.OrganizationRoles  = {EOrganizationRole::Member};
+	InviteUser2.UserEmail		  = AlternativeLoginEmail;
+	InviteUser2.OrganizationRoles = {EOrganizationRole::Member};
 
 	InviteOrganizationRoleCollection OrganizationInvites;
-	OrganizationInvites.EmailLinkUrl = "https://dev.magnoverse.space";
-	OrganizationInvites.SignupUrl = "https://dev.magnoverse.space";
+	OrganizationInvites.EmailLinkUrl	 = "https://dev.magnoverse.space";
```

Comment:
Question - do we think this would be better data-driven, in the same way as account creds? The test assumes the developer is working with magnopus-internal backend environments, which is why I ask.

Reply:
Yes I agree that could/should be data driven. Will require changes to a few tests (at least User System Tests), so I would rather create a separate ticket for that work. You happy with that?

Reply:
Yup!